inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -177,4 +181,93 @@ public final class Constants {
                     .defaultValue(false)
                     .withDescription("Because spark do not support iceberg data type: `timestamp without time zone` and"
                             + "`time`, so type conversions must be mapped to types supported by spark.");
+
+    //========================================= dirty configuration =========================================
+    public static final String DEFAULT_LABEL_IDENTIFER = "dirty-data";
+    public static final String SINK_DIRTY_PREFIX = "sink.dirty.";
+    public static final String DIRTY_PREFIX = "dirty.";
+    public static final String DIRTY_LABEL_DELIMITER = "-";
+
+    public static final ConfigOption<Boolean> DIRTY_IGNORE =
+            ConfigOptions.key("dirty.ignore")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Whether ignore the dirty data, default value is 'false'");
+    public static final ConfigOption<Boolean> DIRTY_SINK_ENABLE =
+            ConfigOptions.key("dirty.sink.enable")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Whether supports dirty data sink, default value is 'false'");
+    public static final ConfigOption<String> SINK_DIRTY_CONNECTOR =
+            ConfigOptions.key("sink.dirty.connector")
```

Comment:
Is there any ambiguity in the description here. 
'sink.dirty.connector'  here describe how dirty data is written, but maybe described how dirty data used in sink node. 
In fact it can be used in source node and sink node.
So maybe 'dirty.sink.xxx' is better?

Reply:
It is a good idea, i will deal with it latter.